spec.html

```diff
@@ -32695,7 +32701,7 @@ <h1>
       <emu-clause id="sec-yearfromtime" type="abstract operation" oldids="eqn-YearFromTime">
         <h1>
           YearFromTime (
-            _t_: a finite time value,
+            _t_: a finite local-offset time value,
```

Comment:
Note that YearFromTime can still be called from MakeDay with a Number that is outside of the local-offset time value range, for example in `Date.UTC(-271821, 3, 20)`. Explanation in https://github.com/tc39/ecma262/issues/1087#issuecomment-1120110847. Same for MonthFromTime and DateFromTime. (This is pre-existing, so doesn't need to block the PR.)